README.md

```diff
@@ -92,3 +92,17 @@ forge --help
 anvil --help
 cast --help
 ```
+
+## Set up graph development
+
+Graphs uses [The Graph](https://thegraph.com/docs).
+
+### Installation
+
+See installation instructions on https://thegraph.com/docs/en/developing/creating-a-subgraph/#install-the-graph-cli.
+
+### Build, test, deploy
+
+- `omen-thumbnailmapping` - see `graphs/omen-thumbnailmapping/package.json`
```

Comment:
What should I run? `codegen` + `build`?

Reply:
Everything. codegen, build, deploy. CLI would guide you if you didn't run something.